pkg/prometheus/operator.go

```diff
@@ -146,7 +146,7 @@ func ValidateRemoteWriteSpec(spec monitoringv1.RemoteWriteSpec) error {
 	return nil
 }
 
-func ValidateAlertmanagerEndpoints(am monitoringv1.AlertmanagerEndpoints) error {
+func ValidateAlertmanagerEndpoints(am monitoringv1.AlertmanagerEndpoints, p monitoringv1.PrometheusInterface) error {
```

Comment:
I commented elsewhere that it could be moved to pkg/prometheus/server. Can you do it in another PR that we'll merge first?

Reply:
Sure, I will make another PR for that change.

Reply:
Would moving it to pkg/prometheus/server/operator.go be appropriate?

Reply:
Sounds good to me!

Reply:
The change has been made in #6480. Could you review?